tests/Fuerte/ConnectionConcurrentTest.cpp

```diff
@@ -146,9 +161,11 @@ TEST_P(ConcurrentConnectionF, CreateDocumentsParallel) {
       }
     });
   }
-  ASSERT_TRUE(wg.wait_for(
+  ASSERT_TRUE(wg->wait_for(
       std::chrono::seconds(300)));  // wait for all threads to return
 
+  guard.fire();
```

Comment:
Should be after join?

Reply:
Otherwise data race on access to connections entry
write in join threads
read here